drivers/SmartThings/harman-luxury/src/api/nsdk.lua

```diff
@@ -88,7 +88,12 @@ local function handleReply(func_name, u, sink, code, valLocationFunc)
     end
     return nil, err
   else -- UNKNOWN VALUE
-    local err = string.format("Error in %s: Unknown return value: code: %s, sink: %s", func_name, code, sink)
+    local err
+    if string.find(code, "timeout") then
```

Comment:
Very minor nit: Since `code` can be both an integer and a string, the type signature on the function doc comments should be `---@param code integer|string`